tests/fake_app.py

```diff
@@ -0,0 +1,20 @@
+from flask import Flask
+
+from flasql.views import GraphQLView
+
+
+def create_graphql_view(**kwargs):
+    schema = kwargs.pop('schema')
+
+    kwargs.update({
+        'schema': schema,
+        'enable_graphiql': True,
+    })
+
+    return GraphQLView.as_view('graphql', **kwargs)
+
+
+def create_app(**kwargs):
+    app = Flask(__name__)
+    app.add_url_rule('/graphql', view_func=create_graphql_view())
+    return app
```

Comment:
Right, but it's the `create_app` in fixtures.py being used, not this one. This one can be deleted best I can tell.